deploy/charts/harvester/values.yaml

```diff
@@ -394,6 +394,8 @@ longhorn:
     ## Specify the concurrent automatic engine upgrade per node limit
     concurrentAutomaticEngineUpgradePerNodeLimit: 3
 
+  defaultStorageClass: true
```

Comment:
Putting a value under `.longhorn` generally means we want to pass the variable to the Longhorn dependency chart. Since the variable is used in the Harvester chart, perhaps moving it to the upper level?